tests/iam_client/iamclient_test.cpp

```diff
@@ -412,7 +414,7 @@ TEST_F(IAMClientTest, NoMemoryOnApplyCertificate)
             EXPECT_EQ(request.cert(), pemCert);
 
             response->set_cert_url(certUrl);
-            response->set_serial(std::string(aos::crypto::cSerialNumSize + 1, 'c'));
+            response->set_serial(std::string(aos::crypto::cSerialNumSize * 3, 'c'));
```

Comment:
we have cSerialNumStrLen in lib_core_cpp aos::crypto namespace

Reply:
Refactored to use cSerialNumStrLen + 1 str size for no memory test case.